Makefile

```diff
@@ -14,10 +14,10 @@ GOFLAGS :=
 dependencies :
 		go get github.com/alecthomas/gometalinter
 		gometalinter --install
-		curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
-		dep ensure
-		@cd vendor/golang.org/x/tools/cmd/goimports; go install .
-		@cd vendor/github.com/onsi/ginkgo/ginkgo; go install .
+		go get golang.org/x/tools/cmd/goimports
+		go get github.com/onsi/ginkgo/ginkgo
+		go mod vendor
+		go mod tidy
```

Comment:
This also seems unnecessary. It could cause unexpected diffs.

Reply:
Hi, @dsharp-pivotal
From the best practise, It's always recommended to run `go mod tidy` after go install or go build. 

Here's some explanation from golang repo:
https://github.com/golang/go/issues/27633

_The reason is that go mod tidy is agnostic of GOOS/GOARCH or build tags; go build is lazy and only does the minimal amount of work required for the current GOOS/GOARCH and build tags._

_cc @bcmills - I wonder whether we should update the documentation for go mod tidy to include some note about this operation being agnostic of GOOS/GOARCH and build tags:_
...
_Because one reading of the documentation is that go build and go mod tidy should leave go.mod in the same state._

Reply:
I kept the `go mod tidy` statement according to above explanation from yangru